### PR TITLE
Change the purpose text for the "Town Square" channel

### PR DESCRIPTION
**Pull request by koddsson**
Note: I have contacted my employer about signing the CLA

#### Summary
Changes the purpose text for the default room, "Town Square".

#### Ticket Link
Fixes [PLT-10999](https://mattermost.atlassian.net/browse/PLT-1099)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/platform/blob/master/i18n/en.json) and [.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) updates

### CODE REVIEW

File: webapp/i18n/en.json [hunk 1]
```diff
@@ -1468,7 +1468,7 @@
   "intro_messages.beginning": "Beginning of {name}",
   "intro_messages.channel": "channel",
   "intro_messages.creator": "This is the start of the {name} {type}, created by {creator} on {date}.",
-  "intro_messages.default": "<h4 class='channel-intro__title'>Beginning of {display_name}</h4><p class='channel-intro__content'><strong>Welcome to {display_name}!</strong><br/><br/>This is the first channel teammates see when they sign up - use it for posting updates everyone needs to know.</p>",
+  "intro_messages.default": "<h4 class='channel-intro__title'>Beginning of {display_name}</h4><p class='channel-intro__content'><strong>Welcome to {display_name}!</strong><br/><br/>This is a special channel for communicating to all members in a team. This channel cannot be deleted and members cannot choose to leave this channel.  Because of this, be careful what you add in here, because everyone will see it</p>",
```
Comment (yangchen1) [inline webapp/i18n/en.json]: Thanks for the PR @koddsson! My comment doesn't have to do with your PR specifically, but more so the ticket - I'm wondering if the last sentence comes off a bit too "red-alert." Will queue for discussion.

**Comment by esethna** [comment]
Hey @koddsson, thanks a lot for this PR! We have discussed in a team meeting that the text should actually be `This is a special channel for communicating to all members in a team. This channel cannot be deleted and members cannot choose to leave this channel. Anything posted here will be seen by everyone.`. Sorry for the confusion on the ticket, totally our fault.

nother note, this text needs to be changed in the [webapp code](https://github.com/mattermost/platform/blob/7df8eaf29aa59467236c1b434bc09ccc33cd011f/webapp/utils/channel_intro_messages.jsx#L136) as well as the localization file.

image

**Comment by esethna** [comment]
Hi @koddsson, just checking in if you need help with the changes requested?

Just so you know, there may be a delay in reviewing and merging this PR as we are locking down feature and improvement contributions for the v3.6 release, per our [release process](https://docs.mattermost.com/process/release-process.html). This PR can be merged into `master` after we branch `release-3.6` on January 4th.

**Comment by jasonblais** [comment]
Hey @koddsson, thank you for this pull request!

Wondering if you've had a chance to contact your employer about signing the CLA?

Also, given the text change is due to our mistake, we can make an update for it as well if you'd prefer.

**Comment by jasonblais** [comment]
@koddsson We really value your interest in contributing this improvement and appreciate that you took the time to create a pull request for it.

However, since we haven't received an update for a few weeks, we will assume that the change is no longer being worked on.

If you'd like to take a second look at the PR or submit a change for another improvement, let us know! Would be happy to work with you :)

**Pull request closed without merging**